kolibri/core/tasks/management/commands/base.py

```diff
@@ -30,19 +33,13 @@ def __init__(self, total=100, level=0, update_callback=None):
         self.level = level
         self.update_callback = update_callback
 
-        # initialize the tqdm progress bar
-        self.progressbar = tqdm(total=total)
-
     def update_progress(self, increment=1, message="", extra_data=None):
-
-        self.progressbar.update(increment)
-
         self.progress += increment
-
         self.message = message
-
         self.extra_data = extra_data
 
+        logging.info('Progress: {:.0%} | {} | {}'.format(1.0 * self.progress / self.total, self.message, self.extra_data))
```

Comment:
Would be good to condition the third element of the progress tracker on not being `None`.

Reply:
I'm happy to remove it entirely

Reply:
Yeah, just not outputting it seems fine as well.

Reply:
(but maybe sys.stdout is a nice compromise)

Reply:
I think we also need to remove this line: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/content/management/commands/importcontent.py#L225 in the import retry logic, since we are not having progressbar attribute anymore.

Reply:
Here's a snippet that I wrote for KA Lite. So because it keeps overwriting the same line, you get the effect of a progress bar:

```
while not done:
    done = do_something()
    fraction = get_progress()
    sys.stdout.write("\rProgress: [{}{}] ({}%)".format(
        "=" * (int(fraction * 100) / 2),
        " " * (50 - int(fraction * 100) / 2),
        int(fraction * 100),
    ))
```

Reply:
Wait so stdout or stderr?

Reply:
is this desired behavior?

does it behave strangely in a multi-threaded environment with multiple processes writing to the same terminal?

Reply:
1. It depends on who you are targeting with the output: A user should see clean output with the progress bar uninterrupted, whilst a dev will understand that it can get a bit messy.
1. To mitigate the clutter in the developers/debuggers log output: Avoid updating the progress bar too often, make a callback that only fires once every other second or even less.